package.json

```diff
@@ -11,7 +11,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "sats-connect": "^1.1.1",
+    "sats-connect": "^1.1.1-d6204de",
```

Comment:
once the sats-connect pr https://github.com/secretkeylabs/sats-connect/pull/40 gets merged in we should update the version number of the sats-connect
@victorkirov  and without the `^` carrot

Reply:
caret carat carrot 😂

Reply:
😂